### PR TITLE
Content Changes for YubiKey 5Ci Launch

### PR DESCRIPTION
**Pull request by iamjake**
These are the required content changes needed for YubiKey 5Ci Launch. They have been reviewed by parts of Yubico Internal Team.

### CODE REVIEW

File: content/Software_Projects/Mobile_SDK/App_Registration_Process/index.adoc [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+Below you will find the guidelines and process for an Apple application developer to register an iOS app for approval that has completed the Yubico iOS SDK integration.
+
+NOTE: This process is not required for Android applications.
+
+=== Approval Process
+image::iOS_App_Review_Process.png[]
+The <<Actions Required By You,Actions Required By You>> section provides detailed instructions for each step in the approval process. Submitting an app to the App Store without the Yubico 5Ci MFi PPID will result in rejection from Apple.
+
+=== UI Components for YubiKey iOS Apps
+Download the https://github.com/Yubico/yubikit-ios[SDK.zip] file for a list of free-to-use example assets for use in your YubiKey-enabled iOS application. Under the extracted folder:
```
Comment (elukewalker) [inline content/Software_Projects/Mobile_SDK/App_Registration_Process/index.adoc]: Should this link to https://github.com/Yubico/yubikit-ios/releases which hold the zip files?

File: content/Software_Projects/Mobile_SDK/App_Registration_Process/index.adoc [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+ - Quality: The app will provide a user experience that does not diminish the users’ perceived quality of the YubiKey and its performance.
+   * Test all features that interact with the YubiKey functionality before registering an app for review. See features below that may be verified by Yubico for quality if applicable.  
+ - Brand: The app must follow the Yubico usage guidelines when presenting the Yubico image or logo.
+   * See https://www.yubico.com/press/images/ for usage guidelines.
+
+
+=== [[Actions Required By You]]Actions Required By You
+
+1. Register App with Yubico - Register your app by filling out our https://partners.yubico.com/prm/English/c/lightning-partner-preview-support/[Yubico App Registration Form]. Include the following:
```
Comment (elukewalker) [inline content/Software_Projects/Mobile_SDK/App_Registration_Process/index.adoc]: Shouldn't this link to /Software_Projects/Mobile_SDK/Register_Your_App/?

File: content/Software_Projects/Mobile_SDK/App_Registration_Process/index.adoc [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+
+a. Apple’s App https://developer.apple.com/app-store/review/[Review Page] and https://developer.apple.com/app-store/review/#common-app-rejections[Common App Rejections Page]
+
+b. https://developer.apple.com/app-store/review/guidelines/[App Store Review Guidelines for iOS Apps]
+
+c. https://developer.apple.com/design/human-interface-guidelines/ios/overview/themes/[Human Interface Guidelines for iOS]
+
+Please note that Yubico reserves the right to remove apps from its whitelist at any time, in our sole discretion, including but not limited to if apps no longer adhere to Yubico’s guidelines.
+
+Contact For questions or comments not covered by this document please open a https://support.yubico.com/support/tickets/newhttps://support.yubico.com/support/tickets/new[developer support ticket].
```
Comment (elukewalker) [inline content/Software_Projects/Mobile_SDK/App_Registration_Process/index.adoc]: It looks like the link here was pasted twice.

File: content/Software_Projects/Mobile_SDK/index.adoc [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+=== Yubico Mobile iOS SDK
+
+The Yubico Mobile iOS SDK is an iOS library provided by Yubico to interact with YubiKeys on iOS devices.
+
+* https://github.com/Yubico/yubikit-ios[Download iOS SDK]
+
+The library supports NFC-enabled YubiKeys and provides the APIs to request an OTP (Yubico OTP or HOTP) from the NFC YubiKeys using a NFC-enabled iOS device. It also provides a built-in QR Code reader which can be used as an alternative enrolment mechanism for iOS devices which don't support NFC reading.
+
+Starting from version 2.0.0, YubiKit adds support for YubiKeys with lightning connector, such as the YubiKey 5Ci, a security key design by Yubico for iOS devices.
```
Comment (elukewalker) [inline content/Software_Projects/Mobile_SDK/index.adoc]: Can we make this the first paragraph?

File: content/Software_Projects/Mobile_SDK/index.adoc [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+
+The library is provided with a demo application which shows a complete example of how to integrate and use all the features of the library in an iOS project.
+
+Ready to get started? Watch our Technical Overview video https://www.youtube.com/watch?v=FpcC-9BLgDA[here] to get an idea of the process from start to finish. Or dive right in and download the SDKs from the links below. If you run into any issues during the process, fill out a developer https://support.yubico.com/support/tickets/new[support ticket] and our team will be able to assist you.
```
Comment (elukewalker) [inline content/Software_Projects/Mobile_SDK/index.adoc]: Remove "Or dive right in and download the SDKs from the links below."

File: content/index.partial [hunk 6]
```diff
@@ -15,6 +15,24 @@
   </div>
 </div>
 
+<!-- Start 5ci -->
+<div id="fiveci" class="row flex">
+  <div class="col-sm-12 col-md-12 col-lg-12">
+    <h1>Getting Started with YubiKey 5Ci</h1>
+    <h3>Yubico SDK for iOS and Android</h3>
+    <h3>Are you developing an iOS or Android app and want to integrate strong authentication support? Now you can with
```
Comment (elukewalker) [inline content/index.partial]: Change "integrate strong authentication support" to "integrate strong authentication"

File: content/index.partial [hunk 7]
```diff
@@ -54,9 +72,13 @@
...
             </a>
-            <a class="img-link" href="/Software_Projects/Mobile%20iOS%20SDK/">
+            <a class="img-link" href="/Software_Projects/Mobile_SDK/">
               <img src="/css/icon-cirtcle-arrow-blue.png" srcset="/css/icon-cirtcle-arrow-blue.png 1x, /css/icon-cirtcle-arrow-blue@2x.png 2x" />
-              Mobile SDK for iOS
+<<<<<<< HEAD
+              Mobile SDK
+=======
+              Mobile SDKs'
+>>>>>>> master
```
Comment (elukewalker) [inline content/index.partial]: Remove
<<<<<<< HEAD
              Mobile SDK
=======
              Mobile SDKs'
>>>>>>> master

File: static/css/main.css [hunk 8]
```diff
@@ -191,14 +191,24 @@ td > p {
   color: white;
 }
 
-#learn h1, #learn h4, #learn ul, #connect h1, #connect ul, #integrate h1, #integrate ul, #open-source h1, #open-source ul {
+<<<<<<< HEAD
```
Comment (elukewalker) [inline static/css/main.css]: Resolve merge issue here

File: static/css/main.css [hunk 9]
```diff
@@ -191,14 +191,24 @@ td > p {
...
 }
 
-#learn h3, #connect h3, #integrate h3, #open-source h3 {
+<<<<<<< HEAD
+#learn h3, #connect h3, #integrate h3, #open-source h3, #fiveci h3 {
+  font-family: "HelveticaNeue", "Open sans", Arial, sans-serif;
+=======
+#5ci h3, #learn h3, #connect h3, #integrate h3, #open-source h3 {
   font-family: 'HelveticaNeue', 'Open sans', Arial, sans-serif;
+>>>>>>> master
```
Comment (elukewalker) [inline static/css/main.css]: Resolve merge issue here

File: content/Software_Projects/Mobile_SDK/App_Registration_Process/index.adoc (hunk 1 above)
Comment (iamjake) [reply content/Software_Projects/Mobile_SDK/App_Registration_Process/index.adoc]: No, per Lan's email the link is correct.

File: content/Software_Projects/Mobile_SDK/App_Registration_Process/index.adoc (hunk 2 above)
Comment (iamjake) [reply content/Software_Projects/Mobile_SDK/App_Registration_Process/index.adoc]: This is what was on the original doc spec, we would need clarity from Lan

File: content/Software_Projects/Mobile_SDK/index.adoc (hunk 4 above)
Comment (iamjake) [reply content/Software_Projects/Mobile_SDK/index.adoc]: We can but would need clarifying from Lan

**Pull request merged**